CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to wolfi-dev/os
+
+* [Setup development environmment](#setup-development-environment)
+
+## Setup development environment
+
+To ease the development of Wofli OS, you can use `chainguard-images/sdk` container image that already include both [apko](https://github.com/chainguard-dev/apko) & [melange](https://github.com/chainguard-dev/melange).
+
+To start a development environment, do
+
+```sh
+make dev-container
+```
+
+What it does is start the `chainguard-images/sdk` image and mount the current working directory into it.
+
+Now, the `Makefile` script assume that there're melange folder & melange binary in there. I change it locally to the following because it's where they are in `chainguard-image/sdk`.
```

Comment:
Where is _in there_?  Try to rephrase this paragraph. I don't fully get what is happening here, but it sounds like a bug we should fix in the scripts.

Reply:
when i use `chainguard-imgaes/sdk`, the melange dir is `/usr/share/melange` and not `../melange` as in the Makefile. that's what i meant.